.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - run: dotnet --info
     - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819
     - if: contains(matrix.runs-on, 'windows')
-      run: build.cmd
+      run: ./build.cmd
```

Comment:
Need to investigate why this was necessary. Seemed to have worked before

Reply:
I have absolutely no experience in tuning CI, I did it by intuition.

Reply:
I think something in github actions was changed. Look at [this PR](https://github.com/adamralph/minver/pull/292/files) - same changes.